sensor_msgs/PointCloud2 original_cloud
sensor_msgs/PointCloud2[] segmented_objects
iri_tos_supervoxels/plane_coefficients plane_coefficients
---
iri_table_clearing_predicates/BlockPredicate[] block_predicates
iri_table_clearing_predicates/BlockGraspPredicate[] block_grasp_predicates
iri_table_clearing_predicates/OnTopPredicate[] on_top_predicates
iri_table_clearing_predicates/PushingDirections[] objects_pushing_directions
iri_table_clearing_predicates/GraspingPoses[] grasping_poses
iri_table_clearing_predicates/PushingPoses[] pushing_poses
iri_table_clearing_predicates/AABB[] aabbs
geometry_msgs/Point[] centroids
iri_table_clearing_predicates/PrincipalDirections[] principal_directions
